### PR TITLE
Fixes console log error on TokenInformation

### LINKED ISSUE #1466: TokenInformation.vue logs warnings in the console during mount
## Description

This means that we are not checking something properly

```
[Vue warn]: Error in render: "TypeError: Cannot read property 'toHexString' of undefined"

found in

---> <TokenInformation> at src/components/TokenInformation.vue
       <VForm>
         <SelectHub> at src/components/navigation/SelectHub.vue
           <LazyRoute> at src/views/LazyRoute.vue
             <SelectHubRoute> at src/views/SelectHubRoute.vue
               <VContent>
                 <VApp>
                   <App> at src/App.vue
                     <Root>
warn @ vue.runtime.esm.js:619
logError @ vue.runtime.esm.js:1884
globalHandleError @ vue.runtime.esm.js:1879
handleError @ vue.runtime.esm.js:1839
Vue._render @ vue.runtime.esm.js:3550
updateComponent @ vue.runtime.esm.js:4066
get @ vue.runtime.esm.js:4479
Watcher @ vue.runtime.esm.js:4468
mountComponent @ vue.runtime.esm.js:4073
Vue.$mount @ vue.runtime.esm.js:8415
init @ vue.runtime.esm.js:3118
createComponent @ vue.runtime.esm.js:5978
createElm @ vue.runtime.esm.js:5925
createChildren @ vue.runtime.esm.js:6053
createElm @ vue.runtime.esm.js:5954
patch @ vue.runtime.esm.js:6477
Vue._update @ vue.runtime.esm.js:3945
updateComponent @ vue.runtime.esm.js:4066
get @ vue.runtime.esm.js:4479
Watcher @ vue.runtime.esm.js:4468
mountComponent @ vue.runtime.esm.js:4073
Vue.$mount @ vue.runtime.esm.js:8415
init @ vue.runtime.esm.js:3118
createComponent @ vue.runtime.esm.js:5978
createElm @ vue.runtime.esm.js:5925
patch @ vue.runtime.esm.js:6477
Vue._update @ vue.runtime.esm.js:3945
updateComponent @ vue.runtime.esm.js:4066
get @ vue.runtime.esm.js:4479
Watcher @ vue.runtime.esm.js:4468
mountComponent @ vue.runtime.esm.js:4073
Vue.$mount @ vue.runtime.esm.js:8415
init @ vue.runtime.esm.js:3118
createComponent @ vue.runtime.esm.js:5978
createElm @ vue.runtime.esm.js:5925
updateChildren @ vue.runtime.esm.js:6216
patchVnode @ vue.runtime.esm.js:6319
patch @ vue.runtime.esm.js:6482
Vue._update @ vue.runtime.esm.js:3948
updateComponent @ vue.runtime.esm.js:4066
get @ vue.runtime.esm.js:4479
run @ vue.runtime.esm.js:4554
flushSchedulerQueue @ vue.runtime.esm.js:4310
eval @ vue.runtime.esm.js:1980
flushCallbacks @ vue.runtime.esm.js:1906
Promise.then (async)
timerFunc @ vue.runtime.esm.js:1933
nextTick @ vue.runtime.esm.js:1990
queueWatcher @ vue.runtime.esm.js:4402
update @ vue.runtime.esm.js:4544
notify @ vue.runtime.esm.js:730
reactiveSetter @ vue.runtime.esm.js:1055
loadComplete @ index.ts:72
wrappedMutationHandler @ vuex.esm.js:771
commitIterator @ vuex.esm.js:407
eval @ vuex.esm.js:406
_withCommit @ vuex.esm.js:551
commit @ vuex.esm.js:405
boundCommit @ vuex.esm.js:350
_callee4$ @ raiden-service.ts:209
tryCatch @ runtime.js:45
invoke @ runtime.js:274
prototype.<computed> @ runtime.js:97
asyncGeneratorStep @ raiden-service.ts:62
_next @ raiden-service.ts:64
Promise.then (async)
asyncGeneratorStep @ raiden-service.ts:62
_next @ raiden-service.ts:64
Promise.then (async)
asyncGeneratorStep @ raiden-service.ts:62
_next @ raiden-service.ts:64
Promise.then (async)
asyncGeneratorStep @ raiden-service.ts:62
_next @ raiden-service.ts:64
Promise.then (async)
asyncGeneratorStep @ raiden-service.ts:62
_next @ raiden-service.ts:64
Promise.then (async)
asyncGeneratorStep @ raiden-service.ts:62
_next @ raiden-service.ts:64
Promise.then (async)
asyncGeneratorStep @ raiden-service.ts:62
_next @ raiden-service.ts:64
eval @ raiden-service.ts:64
eval @ raiden-service.ts:64
connect @ raiden-service.ts:209
_callee$ @ Home.vue:80
tryCatch @ runtime.js:45
invoke @ runtime.js:274
prototype.<computed> @ runtime.js:97
asyncGeneratorStep @ cjs.js?!./node_modules/babel-loader/lib/index.js!./node_modules/ts-loader/index.js?!./node_modules/vuetify-loader/lib/loader.js?!./node_modules/cache-loader/dist/cjs.js?!./node_modules/vue-loader/lib/index.js?!./src/views/Home.vue?vue&type=script&lang=ts&:21
_next @ cjs.js?!./node_modules/babel-loader/lib/index.js!./node_modules/ts-loader/index.js?!./node_modules/vuetify-loader/lib/loader.js?!./node_modules/cache-loader/dist/cjs.js?!./node_modules/vue-loader/lib/index.js?!./src/views/Home.vue?vue&type=script&lang=ts&:23
eval @ cjs.js?!./node_modules/babel-loader/lib/index.js!./node_modules/ts-loader/index.js?!./node_modules/vuetify-loader/lib/loader.js?!./node_modules/cache-loader/dist/cjs.js?!./node_modules/vue-loader/lib/index.js?!./src/views/Home.vue?vue&type=script&lang=ts&:23
eval @ cjs.js?!./node_modules/babel-loader/lib/index.js!./node_modules/ts-loader/index.js?!./node_modules/vuetify-loader/lib/loader.js?!./node_modules/cache-loader/dist/cjs.js?!./node_modules/vue-loader/lib/index.js?!./src/views/Home.vue?vue&type=script&lang=ts&:23
connect @ Home.vue:80
invokeWithErrorHandling @ vue.runtime.esm.js:1854
invoker @ vue.runtime.esm.js:2179
invokeWithErrorHandling @ vue.runtime.esm.js:1854
Vue.$emit @ vue.runtime.esm.js:3888
emit @ vue-property-decorator.js:250
emitter @ vue-property-decorator.js:261
click @ generated-ConnectDialog.vue?8747:62
invokeWithErrorHandling @ vue.runtime.esm.js:1854
invoker @ vue.runtime.esm.js:2179
original._wrapper @ vue.runtime.esm.js:6917
Show 15 more frames
vue.runtime.esm.js:1888 TypeError: Cannot read property 'toHexString' of undefined
    at new BigNumber (bignumber.js:89)
    at Object.bigNumberify (bignumber.js:182)
    at formatUnits (units.js:99)
    at Function.toUnits (balance-utils.ts:9)
    at displayFormat (filters.ts:34)
    at Proxy.render (eval at ./node_modules/cache-loader/dist/cjs.js?{"cacheDirectory":"node_modules/.cache/vue-loader","cacheIdentifier":"484a7714-vue-loader-template"}!./node_modules/vue-loader/lib/loaders/templateLoader.js?!./node_modules/vuetify-loader/lib/loader.js?!./node_modules/cache-loader/dist/cjs.js?!./node_modules/vue-loader/lib/index.js?!./src/components/TokenInformation.vue?vue&type=template&id=005e97ba&scoped=true& (2.js:71), <anonymous>:68:48)
    at VueComponent.Vue._render (vue.runtime.esm.js:3548)
    at VueComponent.updateComponent (vue.runtime.esm.js:4066)
    at Watcher.get (vue.runtime.esm.js:4479)
    at new Watcher (vue.runtime.esm.js:4468)[Vue warn]: Error in render: "TypeError: Cannot read property 'toHexString' of undefined"

found in

---> <TokenInformation> at src/components/TokenInformation.vue
       <VForm>
         <SelectHub> at src/components/navigation/SelectHub.vue
           <LazyRoute> at src/views/LazyRoute.vue
             <SelectHubRoute> at src/views/SelectHubRoute.vue
               <VContent>
                 <VApp>
                   <App> at src/App.vue
                     <Root>
warn @ vue.runtime.esm.js:619
logError @ vue.runtime.esm.js:1884
globalHandleError @ vue.runtime.esm.js:1879
handleError @ vue.runtime.esm.js:1839
Vue._render @ vue.runtime.esm.js:3550
updateComponent @ vue.runtime.esm.js:4066
get @ vue.runtime.esm.js:4479
Watcher @ vue.runtime.esm.js:4468
mountComponent @ vue.runtime.esm.js:4073
Vue.$mount @ vue.runtime.esm.js:8415
init @ vue.runtime.esm.js:3118
createComponent @ vue.runtime.esm.js:5978
createElm @ vue.runtime.esm.js:5925
createChildren @ vue.runtime.esm.js:6053
createElm @ vue.runtime.esm.js:5954
patch @ vue.runtime.esm.js:6477
Vue._update @ vue.runtime.esm.js:3945
updateComponent @ vue.runtime.esm.js:4066
get @ vue.runtime.esm.js:4479
Watcher @ vue.runtime.esm.js:4468
mountComponent @ vue.runtime.esm.js:4073
Vue.$mount @ vue.runtime.esm.js:8415
init @ vue.runtime.esm.js:3118
createComponent @ vue.runtime.esm.js:5978
createElm @ vue.runtime.esm.js:5925
patch @ vue.runtime.esm.js:6477
Vue._update @ vue.runtime.esm.js:3945
updateComponent @ vue.runtime.esm.js:4066
get @ vue.runtime.esm.js:4479
Watcher @ vue.runtime.esm.js:4468
mountComponent @ vue.runtime.esm.js:4073
Vue.$mount @ vue.runtime.esm.js:8415
init @ vue.runtime.esm.js:3118
createComponent @ vue.runtime.esm.js:5978
createElm @ vue.runtime.esm.js:5925
updateChildren @ vue.runtime.esm.js:6216
patchVnode @ vue.runtime.esm.js:6319
patch @ vue.runtime.esm.js:6482
Vue._update @ vue.runtime.esm.js:3948
updateComponent @ vue.runtime.esm.js:4066
get @ vue.runtime.esm.js:4479
run @ vue.runtime.esm.js:4554
flushSchedulerQueue @ vue.runtime.esm.js:4310
eval @ vue.runtime.esm.js:1980
flushCallbacks @ vue.runtime.esm.js:1906
Promise.then (async)
timerFunc @ vue.runtime.esm.js:1933
nextTick @ vue.runtime.esm.js:1990
queueWatcher @ vue.runtime.esm.js:4402
update @ vue.runtime.esm.js:4544
notify @ vue.runtime.esm.js:730
reactiveSetter @ vue.runtime.esm.js:1055
loadComplete @ index.ts:72
wrappedMutationHandler @ vuex.esm.js:771
commitIterator @ vuex.esm.js:407
eval @ vuex.esm.js:406
_withCommit @ vuex.esm.js:551
commit @ vuex.esm.js:405
boundCommit @ vuex.esm.js:350
_callee4$ @ raiden-service.ts:209
tryCatch @ runtime.js:45
invoke @ runtime.js:274
prototype.<computed> @ runtime.js:97
asyncGeneratorStep @ raiden-service.ts:62
_next @ raiden-service.ts:64
Promise.then (async)
asyncGeneratorStep @ raiden-service.ts:62
_next @ raiden-service.ts:64
Promise.then (async)
asyncGeneratorStep @ raiden-service.ts:62
_next @ raiden-service.ts:64
Promise.then (async)
asyncGeneratorStep @ raiden-service.ts:62
_next @ raiden-service.ts:64
Promise.then (async)
asyncGeneratorStep @ raiden-service.ts:62
_next @ raiden-service.ts:64
Promise.then (async)
asyncGeneratorStep @ raiden-service.ts:62
_next @ raiden-service.ts:64
Promise.then (async)
asyncGeneratorStep @ raiden-service.ts:62
_next @ raiden-service.ts:64
eval @ raiden-service.ts:64
eval @ raiden-service.ts:64
connect @ raiden-service.ts:209
_callee$ @ Home.vue:80
tryCatch @ runtime.js:45
invoke @ runtime.js:274
prototype.<computed> @ runtime.js:97
asyncGeneratorStep @ cjs.js?!./node_modules/babel-loader/lib/index.js!./node_modules/ts-loader/index.js?!./node_modules/vuetify-loader/lib/loader.js?!./node_modules/cache-loader/dist/cjs.js?!./node_modules/vue-loader/lib/index.js?!./src/views/Home.vue?vue&type=script&lang=ts&:21
_next @ cjs.js?!./node_modules/babel-loader/lib/index.js!./node_modules/ts-loader/index.js?!./node_modules/vuetify-loader/lib/loader.js?!./node_modules/cache-loader/dist/cjs.js?!./node_modules/vue-loader/lib/index.js?!./src/views/Home.vue?vue&type=script&lang=ts&:23
eval @ cjs.js?!./node_modules/babel-loader/lib/index.js!./node_modules/ts-loader/index.js?!./node_modules/vuetify-loader/lib/loader.js?!./node_modules/cache-loader/dist/cjs.js?!./node_modules/vue-loader/lib/index.js?!./src/views/Home.vue?vue&type=script&lang=ts&:23
eval @ cjs.js?!./node_modules/babel-loader/lib/index.js!./node_modules/ts-loader/index.js?!./node_modules/vuetify-loader/lib/loader.js?!./node_modules/cache-loader/dist/cjs.js?!./node_modules/vue-loader/lib/index.js?!./src/views/Home.vue?vue&type=script&lang=ts&:23
connect @ Home.vue:80
invokeWithErrorHandling @ vue.runtime.esm.js:1854
invoker @ vue.runtime.esm.js:2179
invokeWithErrorHandling @ vue.runtime.esm.js:1854
Vue.$emit @ vue.runtime.esm.js:3888
emit @ vue-property-decorator.js:250
emitter @ vue-property-decorator.js:261
click @ generated-ConnectDialog.vue?8747:62
invokeWithErrorHandling @ vue.runtime.esm.js:1854
invoker @ vue.runtime.esm.js:2179
original._wrapper @ vue.runtime.esm.js:6917
Show 15 more frames
vue.runtime.esm.js:1888 TypeError: Cannot read property 'toHexString' of undefined
    at new BigNumber (bignumber.js:89)
    at Object.bigNumberify (bignumber.js:182)
    at formatUnits (units.js:99)
    at Function.toUnits (balance-utils.ts:9)
    at displayFormat (filters.ts:34)
    at Proxy.render (eval at ./node_modules/cache-loader/dist/cjs.js?{"cacheDirectory":"node_modules/.cache/vue-loader","cacheIdentifier":"484a7714-vue-loader-template"}!./node_modules/vue-loader/lib/loaders/templateLoader.js?!./node_modules/vuetify-loader/lib/loader.js?!./node_modules/cache-loader/dist/cjs.js?!./node_modules/vue-loader/lib/index.js?!./src/components/TokenInformation.vue?vue&type=template&id=005e97ba&scoped=true& (2.js:71), <anonymous>:68:48)
    at VueComponent.Vue._render (vue.runtime.esm.js:3548)
    at VueComponent.updateComponent (vue.runtime.esm.js:4066)
    at Watcher.get (vue.runtime.esm.js:4479)
    at new Watcher (vue.runtime.esm.js:4468)
```

## Acceptance criteria
- 

## Tasks
- [ ]

### PR DESCRIPTION
**Pull request by kelsos**
Fixes #1466  

**Short description**
Fixes a warning logged by TokenInformation.vue when navigating to a lazy route and connecting.

**Definition of Done**

- [x] Steps to manually test the change have been documented
- [x] Acceptance criteria are met
- [ ] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1. Navigate http://localhost:8080/connect/0x59105441977ecD9d805A4f5b060E34676F50F806
2. Open debug tools
3. Press connect
4. No warning on the console check #1466 for the message that should not appear

### CODE REVIEW

File: raiden-dapp/src/components/TokenInformation.vue [hunk 1]
```diff
@@ -29,7 +29,7 @@
       <v-col cols="8">
         <div class="token-information__description">
           <span class="token-information__balance">
-            {{ token.balance | displayFormat(token.decimals) }}
+            {{ (token.balance || 0) | displayFormat(token.decimals) }}
```
Comment (nephix) [inline raiden-dapp/src/components/TokenInformation.vue:32]: `decimals` can be undefined as well iirc? Not sure

Comment (kelsos) [reply raiden-dapp/src/components/TokenInformation.vue:32]: Yes they can but the filter already takes care of that.

Comment (nephix) [reply raiden-dapp/src/components/TokenInformation.vue:32]: Ah ok 👍

Comment (kelsos) [reply raiden-dapp/src/components/TokenInformation.vue:32]: https://github.com/raiden-network/light-client/blob/a48b2259d0a934689b1bd30392cbc72395866f18/raiden-dapp/src/filters.ts#L34

**Pull request merged**